webpack.config.babel.js

```diff
@@ -78,11 +78,11 @@ const appConfig = {
       {
         // Proxy all server-served routes:
         context: ['/auth', '/img', '/css', '/ckeditor', '/video', '/api', '/definitions'],
-        target: 'http://localhost:9001',
+        target: process.env.APP_URI,
       },
     ],
     compress: false,
-    port: 9000,
+    port: process.env.WEB_APP_PORT,
```

Comment:
Can we have a default to 9000 so that we don’t necessarily need to add it to .env ?

Reply:
Makes sense, added it.